easybuild/framework/extensioneasyblock.py

```diff
@@ -81,6 +81,7 @@ def __init__(self, *args, **kwargs):
             # name and version properties of EasyBlock are used, so make sure name and version are correct
             self.cfg['name'] = self.ext.get('name', None)
             self.cfg['version'] = self.ext.get('version', None)
+            self.cfg['start_dir'] = self.ext.get('options', {}).get('start_dir', None)
```

Comment:
Do you have an example easyconfig where you'd like to use this?

Since #2194, I don't think this is actually needed?

Reply:
Why would it work? The `prepare_step` is never called if you install an extension under the `extensions_step`? 

Current use case: arrow. It has the main C++ library in a subdirectory that needs to be build and installed first. Afterwards you go to the python subdirectory to build the python extensions.
See: easybuilders/easybuild-easyconfigs#5416

Reply:
OK, now I understand better what the issue is here, you need to prevent that the `start_dir` value is inherited from the parent, where it's usually defined...

Can you clarify that with a comment above this line?